.travis.yml

```diff
@@ -36,6 +36,26 @@ addons:
       - libprotobuf-dev
       - protobuf-compiler
 
+  coverity_scan:
+
+    project:
+      name: tiny-dnn/tiny-dnn
+      version: 1.0
+      description: Header only, dependency-free deep learning framework in C++11
+
+    notification_email: example@example.com
+
+    build_command_prepend: cmake -DUSE_TBB=ON
+            -DUSE_SSE=ON
+            -DUSE_AVX=ON
+            -DUSE_DOUBLE=OFF
+            -DBUILD_TESTS=ON
+            -DBUILD_EXAMPLES=ON .
+
+    build_command: make -j8
+
+    branch_pattern: master
```

Comment:
https://scan.coverity.com/travis_ci:
> Pattern to match selecting branches that will run analysis. We recommend leaving this set to 'coverity_scan'.
Take care in resource usage, and consider the build frequency allowances per
https://scan.coverity.com/faq#frequency